helpers/setup.sh

```diff
@@ -148,6 +150,8 @@ if [[ "$IS_UPDATE" == "0" ]]; then
     gcloud iam service-accounts keys create "${KEY_FILE}" \
         --iam-account "${SERVICE_ACCOUNT_EMAIL}" \
         --user-output-enabled false
+
+    export GOOGLE_APPLICATION_CREDENTIALS="${KEY_FILE}"
```

Comment:
[This PR](https://github.com/forseti-security/terraform-google-forseti/pull/357/files#diff-280aa428a4c3b4093742fcb0234f5fe8L122) also caused an issue with the tutorial because terraform is expecting to get the credentials from this env var.